themes/makedeb/static/js/theme.js

```diff
@@ -0,0 +1,56 @@
+initialThemeLoad();
+
+const toggleThemeBtn = document.getElementById("toggle-theme-button");
+toggleThemeBtn?.addEventListener("click", function () {
+  const body = document.body;
+  if (body.className == "dark") {
+    setBodyClass("");
+    toggleThemeBtn?.classList.remove("active");
+    changeBtnIcon("moon");
+    localStorage.removeItem("theme");
+  } else {
+    toggleThemeBtn?.classList.add("active");
+    changeBtnIcon("sun");
+    localStorage.setItem("theme", "dark");
+    setBodyClass("dark");
+  }
+});
+
+/**
+ * Makes nessesary changes for initial theme load
+ */
+function initialThemeLoad() {
+  const toggleThemeBtn = document.getElementById("toggle-theme-button");
+  const theme = localStorage.getItem("theme");
+  if (theme == "dark") {
+    setBodyClass("dark");
+    changeBtnIcon("sun");
+    toggleThemeBtn?.classList.add("active");
+  } else changeBtnIcon("moon");
+}
+
+/**
+ * Changes the body class
+ * @param {string} className - class name to be set
+ */
+function setBodyClass(className) {
+  const body = document.body;
+  body.className = className;
+}
```

Comment:
Can't this just be written in one line as `document.body.className = blahBlah` and then put directly in the `toggleThemeBtn?.addEventListener` callback above? Since that's how things like `toggleThemeBtn?.classList.remove` are being used I think that'd make the code more consistent.